components/TransactionDetails/TransactionDetails.styled.js

```diff
@@ -1,11 +1,15 @@
 const { default: styled } = require("styled-components");
 
-export const DetailsContainer = styled.article`
+export const DetailsContainer = styled.div`
+display: ${(props) => props.$showModal ? "none" : "block"}; `
```

Comment:
in the other component you used the same logic with props.$visible. Please use this here as well to keep the code consistent. You can pass showModal to the container like this: 
```jsx
<DetailsContainer $visible={!showModal}> ...
```